### PR TITLE
tweak(docs/building): add note when using python 3.12+

### PR DESCRIPTION
**Pull request by AvarianKnight**
- Python deprecated distutils in [PEP 632](https://peps.python.org/pep-0632/) and removed it in 3.12

### CODE REVIEW

**Comment by thorium-cfx** [comment]
Could you put this one back to [2912c9c](https://github.com/citizenfx/fivem/commit/2912c9c93386df977580eb87f42ff1f51522dbbb)?

**Comment by AvarianKnight** [comment]
> If you're using Python 3.12 or higher you may need to install setuptools when it isn't installed already

The wording here seems a bit redundant does it not?

> If you're using Python 3.12 or higher you may need to install setuptools

**Comment by thorium-cfx** [comment]
True, your proposal looks good 👍

File: docs/building.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ To build FiveM, RedM or FXServer on Windows you need the following dependencies:
   - .NET desktop environment
   - Desktop development with C++
   - Universal Windows Platform development
-
```
Comment (thorium-cfx) [inline docs/building.md:11]: Please no unnecessary changes like these, this'll make git blame look wild

**Review by thorium-cfx — CHANGES_REQUESTED** [review]

**Comment by AvarianKnight** [comment]
Every editor I have force removes redundant white spaces 😢

**Review by thorium-cfx — APPROVED** [review]
